web/src/Web.App/Views/SchoolSpending/_Costs.cshtml

```diff
@@ -0,0 +1,45 @@
+@using Web.App.Extensions
+@using Newtonsoft.Json
+@using Web.App.ViewModels
+@using Web.App.Domain
+
+@model CostsViewModel
+
+@if (Model.Costs.Any())
+    {
+        <a id=@Model.Id></a>
```

Comment:
Empty DOM element is generally not a good idea. Perhaps move the `id` attribute to the `<div>` on line `23` if this is the first `category` in the loop?